### ISSUE TITLE
Keyboard shortcuts on macOS: consider using words in addition to the icons

### ISSUE THREAD

**Issue by jeherve** [WordPress/gutenberg#40721]
## What problem does this address?

Keyboard shortcuts are very useful in an editor like the block editor; they allow you to move faster and to keep focussing on writing content instead of having to stop typing and use a mouse whenever you want to do things like deleting blocks or adding new blocks before or after an existing one:

<img width="322" alt="Screenshot 2022-04-29 at 11 11 28" src="https://user-images.githubusercontent.com/426388/165918498-9f3ec4b3-98a3-4c21-a33b-0a7fd4c8c160.png">

Interestingly, even after having used the block editor as my main editor for years now, I still do not know what most of the shortcuts are. I see them in the editor, and I am used to the macOS interface, but it's still hard for me to decypher things like ⌃⌥Z. I'm not immediately familiar with those icons (even though they are used in the macOS menus and UI), and I have to pause and think to figure out what they mean (Ctrl + Option + Z in this case).

## What is your proposed solution?

Until #9429, we had words such as Ctrl, Cmd, Shft next to those icons, or instead of those icons. I think it would help to have both the words and the icons if possible, to make shortcuts fast to read, and thus increase their adoption.

@jasmussen What's your take on this?

**Comment by jasmussen** [comment]
Seems worth a PR just to get a feel for it! I think we already have text instead of unicode symbols on Windows, as the actual symbols are rarely printed on the keyboard keys. (Though just personally, and specifically for ⌥ and ⌘, I find those more intuitive than Option and CMD).

The tricky part here is managing prominence! These keyboard shortcuts are permanently visible in the menu, which is nice, but there's also the separate keyboard shortcuts sheet in the ellipsis menu. The longer the keyboard shortcut reads aloud, the more visual weight it gets in the menu, which might both make it feel too prominent, and also make it harder to scan / less legible. For example, `Remove Paragraph 	 Ctrl Opt Z` — might work fine?

**Comment by mtias** [comment]
I don't think we should try to deviate from how the OS presents shortcuts here.

<img width="810" alt="image" src="https://user-images.githubusercontent.com/548849/169697427-c80ec44b-71a3-44da-957e-17c14466d8b2.png">
